src/library/scala/collection/Set.scala

```diff
@@ -174,27 +174,24 @@ trait SetOps[A, +CC[_], +C <: SetOps[A, CC, C]]
     *  @param that     the collection containing the elements to add.
     *  @return a new $coll with the given elements added, omitting duplicates.
     */
-  def concat(that: collection.Iterable[A]): C = fromSpecificIterable(new View.Concat(toIterable, that))
+  def concat[B >: A](that: collection.Iterable[B]): CC[B]
```

Comment:
Yes, but I don't think that's necessary since this is already how it's implemented in `IterableOps`. The def here is only useful for the Set-specific documentation comment.